src/frontends/pytorch/src/utils.cpp

```diff
@@ -13,6 +13,16 @@ namespace ov {
 namespace frontend {
 namespace pytorch {
 
+void num_inputs_check(NodeContext& context, int min_inputs, int max_inputs) {
```

Comment:
```suggestion
void num_inputs_check(const NodeContext& context, size_t min_inputs, size_t max_inputs) {
```

Reply:
Additionally, we can check type of the operation in this routine. This check will prevent us from wrong translator call for some operation.